inbm/trtl/btrfs/delete.go

```diff
@@ -23,3 +23,16 @@ func DeleteSnapshot(cw util.ExecCommandWrapper, configName string, snapshotNumbe
 
 	return nil
 }
+
+// DeleteConfig will use Snapper to delete the configuration for a filesystem and subvolume.
+func DeleteConfig(cw util.ExecCommandWrapper, configName string) error {
+	args := []string{"-c", configName, "delete-config"}
```

Comment:
Before deleting a Snapper configuration, you need to delete the associated snapshots first, or else the deletion will fail.

Reply:
(but since it looks like we use rootConfig for all our snapshots, we should only be deleting one snapshot under that config. e.g. if we want to use snapshot '2' we should delete '2' before trying to take that snapshot. that would fix the bug this PR is addressing.)